eidas_node/connector/urls.py

```diff
@@ -0,0 +1,10 @@
+"""URLs of eidas_node.connector."""
+from django.urls import path
+
+from eidas_node.connector import views
+
+urlpatterns = [
+    path('CitizenCountrySelector', views.CitizenCountrySelectorView.as_view(), name='citizen-country-selector'),
```

Comment:
That's very fortunate, because this is actually PascalCase ;-).

Reply:
I suppose actual URL path should be configuration option. Sometimes this information requires to be configured in remote systems and it may not be easy to quickly change it. It's better to have system that can easily adopt to this situation.

Reply:
You can always provide your own `urls.py` if you need to configure it. This should be the "good for almost everyone" case.

Reply:
Path and corresponding names shortened to `CountrySelector`.